docs/modules/integrations/pages/azure-devops-exporter.adoc

```diff
@@ -47,6 +47,25 @@ Please make sure that the `bdd.configuration.formats` property includes JSON val
 |`AUTOMATED` or `MANUAL`
 |The scenario part to export into `Steps` section of `Test Case` work item, the remaining scenario part is exported into `Summary` section.
 
+|`azure-devops-exporter.test-run.name`
+|
+|string
+a|The test run name.
+
+Setting of this property leads to creation of a new test run with the specified name.
```

Comment:
what'll happen if property is not set?

Reply:
the opposite, new test run will not be created :)

Reply:
discussed to add boolean `create` flag